python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S2092.html

```diff
@@ -0,0 +1,62 @@
+<p>When a cookie is protected with the <code>secure</code> attribute set to <em>true</em> it will not be send by the browser over an unencrypted HTTP
+request and thus cannot be observed by an unauthorized person during a man-in-the-middle attack. </p>
+<h2>Ask Yourself Whether</h2>
+<ul>
+  <li> the cookie is for instance a <em>session-cookie</em> not designed to be sent over non-HTTPS communication. </li>
+  <li> it's not sure that the website contains <a href="https://developer.mozilla.org/fr/docs/S%C3%A9curit%C3%A9/MixedContent">mixed content</a> or
+  not (ie HTTPS everywhere or not) </li>
+</ul>
+<p>You are at risk if you answered yes to any of those questions.</p>
+<h2>Recommended Secure Coding Practices</h2>
+<ul>
+  <li> It is recommended to use <code>HTTPs</code> everywhere so setting the <code>secure</code> flag to <em>true</em> should be the default behaviour
+  when creating cookies. </li>
+  <li> Set the <code>secure</code> flag to <em>true</em> for session-cookies. </li>
+</ul>
+<h2>Sensitive Code Examples</h2>
+<p>http.cookies</p>
+<pre>
+import http.cookies
+
+cookie = http.cookies.SimpleCookie()
+cookie['key'] = 'value' # Sensitive
```

Comment:
If I understood correctly, we won't cover this case at first, so I guess it might be worth updating the rule description.